### ISSUE TITLE
DONT DOWNLOAD

### ISSUE THREAD

**Issue by hozzywozzy** [Matrixdevs/Zerologger#1]
THIS IS A VIRUS. [Triage Report](https://tria.ge/230213-t9kpvadh9x/behavioral1)
**VIRUS**
![image](https://user-images.githubusercontent.com/87353619/218519821-c8025f7c-92f1-4d2a-a0ce-cfbd55b35ad8.png)
![image](https://user-images.githubusercontent.com/87353619/218519974-d9e7437a-13c9-43da-87bb-a6f898459318.png)
![image](https://user-images.githubusercontent.com/87353619/218520044-7a8aff0a-1d09-4b62-9650-75cae7e6eaf1.png)
**VM FAKE BLUESCREEN**
![image](https://user-images.githubusercontent.com/87353619/218520344-b4a48904-8c48-4d2a-80f0-cb496d19846e.png)
![image](https://user-images.githubusercontent.com/87353619/218520379-0e45fdf4-f832-4baf-bfdb-729fe678716c.png)
![image](https://user-images.githubusercontent.com/87353619/218520425-6ead13b7-41f1-4720-a8eb-a511a9a8d60f.png)
![image](https://user-images.githubusercontent.com/87353619/218520456-98645647-d263-459d-b4b2-04ab87005f9a.png)

TL:DR
DONT DOWNLOAD THIS IT IS A VIRUS AND WILL STEAL YOUR INFORMATION.

**Comment by hozzywozzy** [comment]
![image](https://user-images.githubusercontent.com/87353619/218522244-d54ae5c8-a361-4378-bb81-697377fc6c43.png)

**Comment by hozzywozzy** [comment]
[37270a1c275dbe482682a628e1d051b5cbd50fac63b6e3ca6c0ce654a335df53.pdf](https://github.com/Matrixdevs/Zerologger/files/10724470/37270a1c275dbe482682a628e1d051b5cbd50fac63b6e3ca6c0ce654a335df53.pdf)

**Comment by hozzywozzy** [comment]
![image](https://user-images.githubusercontent.com/87353619/218523364-1a2d9f7e-1848-4220-ac3d-4befc07a6ecb.png)
Miner
